src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/time.tsx

```diff
@@ -7,33 +7,65 @@ import Tooltip from 'app/components/tooltip';
 import getDynamicText from 'app/utils/getDynamicText';
 import TextOverflow from 'app/components/textOverflow';
 
-const getBreadcrumbTimeTooltipTitle = (timestamp: string) => {
+const getTooltipTitle = (
+  timestamp: string,
+  relativeTime: string,
+  displayRelativeTime?: boolean
+) => {
   const parsedTimestamp = moment(timestamp);
-  const timestampFormat = parsedTimestamp.milliseconds() ? 'll H:mm:ss.SSS A' : 'lll';
-  return parsedTimestamp.format(timestampFormat);
+  const date = parsedTimestamp.format('ll');
+
+  if (!displayRelativeTime) {
+    const formattedTimestamp = moment(timestamp).format('ll H:mm');
+    const formattedRelativeTime = moment(relativeTime).format('ll H:mm');
+    // TODO(Priscila): check if it's possible to replace 'few seconds ago' with 'now'
+    return {date, time: moment(formattedTimestamp).from(formattedRelativeTime)};
```

Comment:
We could say something like if it's <= 60 seconds we consider it "now"

Reply:
sure! I have introduced this method:

```
const fromOrNow = (time: string, timeToCompareWith: string) => {
  if (
    !(
      moment(time).format('HH:mm:ss') === moment(timeToCompareWith).format('HH:mm:ss') &&
      // compares if it happened less than 1 m ago (60000 equals 1m)
      Math.abs(moment(time).diff(timeToCompareWith)) < 60000
    )
  ) {
    return moment(time).from(timeToCompareWith);
  }

  return t('Now');
};
```

Reply:
not sure if I should add it in utils

Reply:
60 seconds was a bit arbitrary from me, are there lower thresholds that would make sense? (i.e. when does the relative time go from `from seconds ago` to `15 seconds ago`)

Reply:
yes if you do  `moment.relativeTimeThreshold('s')` you will get `44 seconds`. According to the docs, the second's range is from 0 - 44.

But even if the difference between 2 timestamps is of only 2000 milliseconds, the `moment.from` method still returns "a few seconds ago" instead of "2 seconds ago".

I've pushed a temp solution until I have time to debug the library and maybe create an issue.

now the UI will display "now" if timeDiff is equal 0 and in case of > 0 and < 60000 it displays "x seconds ago"